mirte_robot/robot.py

```diff
@@ -1,41 +1,94 @@
 #!/usr/bin/env python
+import os
+import platform
 import time
-import rospy
-import rosservice
-import signal
-import sys
 import math
-import atexit
+from typing import TYPE_CHECKING, Optional
+
+import rclpy
+import rclpy.node
+from rclpy.validate_namespace import validate_namespace
+
+if TYPE_CHECKING:
+    import rclpy.client
+
+from controller_manager_msgs.srv import SwitchController
+from mirte_msgs.srv import (
```

Comment:
Is there a specific reason for not importing *

Reply:
Linter warnings, it is generally advised to not `* import`